e2e/cypress/integration/messaging/collapse_link_spec.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Link preview - Removing it from my view removes it from other user\'s view', () => {
+    before(() => {
+        // # Set the configuration on Link Previews
+        setSystemConsoleLinkPreview();
+        setAccountSettingsLinkPreview('user-1');
+        setAccountSettingsLinkPreview('sysadmin');
+
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18708-Link preview - Removing it from my view removes it from other user\'s view', () => {
+        const message = 'https://www.bbc.com/news/uk-wales-45142614';
+
+        // # Create new DM channel with user's email
+        cy.apiGetUsers(['user-1', 'sysadmin']).then((userResponse) => {
+            const userEmailArray = [userResponse.body[1].id, userResponse.body[0].id];
+
+            cy.apiCreateDirectChannel(userEmailArray).then(() => {
+                cy.visit('/ad-1/messages/@sysadmin');
+
+                // # Post message to use
+                cy.postMessage(message);
+
+                cy.getLastPostId().then((postId) => {
+                    // * Check the preview is shown
+                    cy.get(`#${postId}_message`).find('.file-preview__button').should('exist');
+
+                    // # Log-in to the other user
+                    cy.apiLogin('sysadmin');
+                    cy.visit('/ad-1/messages/@user-1');
```

Comment:
```suggestion
                    cy.visit('/');
```
Same case for other instance.